parcel.go

```diff
@@ -13,48 +15,91 @@ func NewParcelStore(db *sql.DB) ParcelStore {
 }
 
 func (s ParcelStore) Add(p Parcel) (int, error) {
-	// реализуйте добавление строки в таблицу parcel, используйте данные из переменной p
-
-	// верните идентификатор последней добавленной записи
-	return 0, nil
+	query := "INSERT INTO parcel (client, status, address, created_at) VALUES (?, ?, ?, ?)"
+	result, err := s.db.Exec(query, p.Client, "registered", p.Address, time.Now().Format(time.RFC3339))
+	if err != nil {
+		return 0, err
+	}
+	id, err := result.LastInsertId()
+	if err != nil {
+		return 0, err
+	}
+	return int(id), nil
 }
 
 func (s ParcelStore) Get(number int) (Parcel, error) {
-	// реализуйте чтение строки по заданному number
-	// здесь из таблицы должна вернуться только одна строка
-
-	// заполните объект Parcel данными из таблицы
-	p := Parcel{}
-
+	query := "SELECT number, client, status, address, created_at FROM parcel WHERE number = ?"
+	var p Parcel
+	err := s.db.QueryRow(query, number).Scan(&p.Number, &p.Client, &p.Status, &p.Address, &p.CreatedAt)
+	if err != nil {
+		if err == sql.ErrNoRows {
```

Comment:
Лучше так всегда делать
```suggestion
		if errors.Is(err, sql.ErrNoRows) {
```